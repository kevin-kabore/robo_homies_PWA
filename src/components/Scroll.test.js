import { shallow } from 'enzyme';
import React from 'react';

import Scroll from './Scroll';

let wrapper;
beforeEach(() => {
	wrapper = shallow(<Scroll />);
})

describe('<Sroll />', () => {
	it('Will render the component', () => {
		expect(wrapper).toMatchSnapshot();
	})
	it('Will have an overflow property of sroll', () => {
		expect(wrapper.prop('style').overflow).toEqual('scroll')
	})
})