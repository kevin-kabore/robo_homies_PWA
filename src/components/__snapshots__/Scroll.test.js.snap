// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sroll /> Will render the component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Scroll />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": undefined,
      "style": Object {
        "border": "1px solid black",
        "height": "800px",
        "overflow": "scroll",
      },
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
        "style": Object {
          "border": "1px solid black",
          "height": "800px",
          "overflow": "scroll",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
